### ISSUE TITLE
Hive Waterloo

### ISSUE THREAD

**Issue by simonaramkisson** [MozillaHive/HiveGlobal#12]
Decision Maker - @chrislarry33
Driver/Owner - @simonaramkisson
Consulted - @ldecoursy
Informed- @mcdermott415

**Comment by simonaramkisson** [comment]
Main Contact: Stephanie Rozek

Stephanie is part of the team that is leading Hive Waterloo but is the most heavily involved and now has received 250,000 in funding from Communitech (the tech hub in Waterloo) to bring the Year of Code to Waterloo

Main Documents:
• https://drive.google.com/a/mozillafoundation.org/?tab=mo#folders/0BwFvBgLGaAgaS3ZRNE1FQzZ5d2s
• https://drive.google.com/a/mozilla.com/file/d/0BwFvBgLGaAgaRk4xUl9OOU9ya00/view?usp=sharing

**Comment by simonaramkisson** [comment]
Call booked with Stephanie on April 1st to discuss the following details:
• complete list of current funders or interested funding contributors for her Year of Code programming
• actual amount of current funds and projected number of funds that will be directed to the YOC program in Waterloo
• Name of team members and setting up a Skype meeting so I can meet everyone on team
• timeline for next 6 months and how will Hive be apart of this timeline
•  Next steps i.e community meeting, identifying community members

**Comment by simonaramkisson** [comment]
Stephanie and Simona are meeting at Mozilla tomorrow to discuss her May 11th meeting with the Kitchener-Waterloo Community Fund.

**Comment by simonaramkisson** [comment]
Stephanie has committed to hosting a Maker Party event, we worked out a timeline where we could see a soft launch of Hive Waterloo. 

Short list of what is needed currently in Waterloo to help their Year of Code project which will funnel into full time Hive Waterloo: 
Short list of what we need:
    Content/curriculum
    Instructors
    Marketing channel
    Assistance in receiving funding

More details on funding:
    We have a verbal commitment for a $100k donation; it must be received as a charitable donation
    Thus we need some facility as an NFP, to partner with a charity in order to issue charitable receipts
    2 possibilities: YoCWR incorporates on its own as a NFP or Mozilla can act as the NFP and flow $ through to the YoC campaign
    We have a meeting April 15 with the KW Community Foundation who may be able to act as a charity in this way - I will keep you posted.

**Comment by simonaramkisson** [comment]
Update:

Waterloo is possibly going to be working Tides Canada as a charitable partner to Year of Code (Waterloo), they would like their projects to be long-term which is why Hive Waterloo is a great step.

E-mail from Stephanie:
Secondly, this is actually an update for you on Tides - I managed to actually speak to someone there yesterday, and now am connected to Nicole Cushing who is looking into things. One of their requirements seems to be that projects they engage with are somewhat longer term, and so I think for them to accept us they would also want to have the confirmation that we will be continuing our activities here in Waterloo past the Year of Code. 

http://tidescanada.org/

Hive Waterloo is ready to launch with this commitment from Tides who will manage and distribute the funds for their 100,000 verbal commitment from the their private donor. 

tl/dr: Hive Waterloo has a charitable partner to accept the 100,000 from a private donor. Tides Canada wants to also commit funding for long-term programming. Stephanie and I believe that Hive Waterloo officially launching satisfies this long-term component. They are already committed to throwing 2 Maker Parties. Hive Waterloo has great potential to be next Hive Learning Network.

**Comment by chrislarry33** [comment]
This awesome...and definitive next steps for us to help this along?

**Comment by simonaramkisson** [comment]
@chrislarry33 Stephanie has a meeting next week (tbd) with Tides Canada and they will discuss the official next steps. Stephanie is coming to Toronto and we are going to do another strategy meeting to discuss how this pot of money is going to fund Year of Code and then lead to Hive Waterloo long-term programming. All Maker Parties will be held under Hive Waterloo brand. Right now we are waiting for Tides to lay out their terms and how they want to form their charitable partnership with Year of Code and how they envision Hive Waterloo being the long term platform. Will update when I get that info.

**Comment by chrislarry33** [comment]
@simonaramkisson Thanks, this is very promising. Lets talk once they get the details with Tides figured out

**Comment by simonaramkisson** [comment]
Will do

**Comment by mcdermott415** [comment]
Do they need/want any prep support from us for convo or planning with Tides?

**Comment by simonaramkisson** [comment]
Launch Update:

Year of Code Waterloo/Hive Waterloo has secured Capacity Canada as a community partner to accept private donation and they are currently working on local grant opportunities: https://capacitycanada.ca 

The main next steps with her team will be to discuss confirming timeline and Maker Party interaction (@amirad, @thornet) and how community meet-ups will be structured, membership process. Current strategy is to engage community organizations supporting YOCW to then lead and support building of Hive community. Waterloo is has a unique opportunity because they already have confirmed support from Waterloo District School Board and Communitech (hub for community tech organizations, digital education and industry)

Stephanie and I have a call with Communitech (Kitchener tech incubator) the week of June 25 (tbc)

**Comment by simonaramkisson** [comment]
Year of Code Waterloo launched this past Saturday and it was a great success. Stephanie will be in Toronto on Wednesday to have a lunch strategy planning session with her team. Will be hosting in Mozilla Toronto and we will be discussing Maker Party plans. http://www.yocwr.ca/event/worldwide-maker-party/

Press Release: https://drive.google.com/drive/u/0/folders/0BwFvBgLGaAgaRVlUM0dESXJlVG8

**Comment by simonaramkisson** [comment]
Stephanie Rozek is coming to Mozfest as the Hive Toronto invite, she has confirmed her travel and will be part of the small meet up of emerging Hive communities.

**Comment by simonaramkisson** [comment]
Simona, Christa Perez and Stephanie meeting in Toronto on January 20 to discuss Hive Waterloo for the first 6 months of 2016, connected with Helen in London regarding how to strategize on better Canadian city connection

**Comment by simonaramkisson** [comment]
Call with Stephanie Rozek and Christa Perez: 
• Hive Waterloo is going to be incorporated as a not for profit in the next 6 months and will change it's name to Hive Waterloo-Region 
• Their next large scale event will be the Family Day HackJam on February 15. Simona will be speaking at the event to discuss Hive Waterloo-Region and be travelling to Waterloo on that day
• The team is currently working on their new website and would like to use the Wordpress template, completion date end of May
• Libra is providing funding for launch of the first week 6 week launch mentoring digital literacy skills, post secondary for young girls 
• Year of Code Waterloo will be the first 6 week pilot, Hive Waterloo Region will own the next 6 week program
• International Women's Day Sunlife Financial as a Sponsor, March 10 will be the event in Waterloo where local politicians to make a pledge to foster better inclusivity 
• He for She Campaign, U of Waterloo is one of the 10 universities in the world participating and Year of Code Waterloo is going to supporting their pledge and events 
• Guinness World Record in June, University of Waterloo and Conestoga providing space- most people to code at one time

**Comment by simonaramkisson** [comment]
Press Release for HackerGrrlz Program:

Year of Code Waterloo Region hits “go’’ button on  HackerGrrlz projects for 2016
http://www.yocwr.ca/hackergrrlz-projects-launch-spring-2016/

Currently up to 50% of women are leaving the tech industry, creating a challenge for employers to retain staff and companies looking to increase employee diversity in order to drive innovation and remain competitive. 

We are looking for employers and women in Waterloo Region who wish to join the program for 2 pilot sessions this spring and already have a growing list of women, employers and parents interested. 

Stephanie Rozek, program director for Year of Code Waterloo Region, is readily available for interviews. You can reach by email stephanie@yocwr.ca or simply reply to this message.
